### PR TITLE
[Sim-Swap-Subscription]: Add TerminationReason SUBSCRIPTION_DELETED

### LINKED ISSUE #172: Termination Reason SUBSCRIPTION_DELETED
**Problem description**
Hi all, I noticed that as of now there is no Termination Reason defined for the case of the User deleting his subscription. 

**Possible evolution**
I would propose to add the Termination Reason:

`SUBSCRIPTION_DELETED - Subscription was deleted by the requester`

### PR DESCRIPTION
**Pull request by daniel-dierich**
#### What type of PR is this?

Add one of the following kinds:

* enhancement/feature

#### What this PR does / why we need it:

Adds TerminationReason SUBSCRIPTION_DELETED

#### Which issue(s) this PR fixes:

Fixes #172

#### Changelog input

```
 release-note

```

#### Additional documentation 

This section can be blank.

```
docs

```

### CODE REVIEW

**Review by bigludo7 — APPROVED** [review]
LGTM

**Review by fernandopradocabrillo — APPROVED** [review]

**Review by bigludo7 — CHANGES_REQUESTED** [review]
Change version to wip lines 64 & 71

File: code/API_definitions/sim-swap-subscriptions.yaml [hunk 1]
```diff
@@ -61,14 +61,14 @@ info:
...
-  version: 0.1.1
+  version: wip
   x-camara-commonalities: 0.4.0
 
 externalDocs:
   description: Product documentation at CAMARA
   url: https://github.com/camaraproject/
 servers:
-  - url: '{apiRoot}/sim-swap-subscriptions/v0.1'
+  - url: '{apiRoot}/sim-swap-subscriptions/wip'
```
Comment (fernandopradocabrillo) [inline code/API_definitions/sim-swap-subscriptions.yaml:71]:
```suggestion
  - url: '{apiRoot}/sim-swap-subscriptions/vwip'

```

**Review by fernandopradocabrillo — APPROVED** [review]
LGTM

**Pull request merged**